src/main/java/com/twilio/sdk/resource/InstanceResource.java

```diff
@@ -153,6 +153,16 @@ public Integer getPropertyAsInteger(String name) {
 	}
 
 	/**
+	 * Gets the property as a Boolean value.
+	 *
+	 * @param name property name
+	 * @return Boolean value
+	 */
+	public Boolean getPropertyAsBoolean(String name) {
+		return Boolean.valueOf(getProperty(name));
```

Comment:
The method being tested is getPropertyAsBoolean and all I care about is, the different possible values getProperty returns, which is anyways covered by the tests. 

If I were to test getProperty, I would follow the steps similar to what you mentioned. Right now testing getProperty it is not in the scope of this PR.